src/ssl.c

```diff
@@ -11057,7 +11057,24 @@ WOLFSSL_SESSION* wolfSSL_get_session(WOLFSSL* ssl)
     return NULL;
 }
 
-
+/*
+ * Sets the session object to use when establishing a TLS/SSL session using
+ * the ssl object. Therefore, this function must be called before
+ * wolfSSL_connect. The session object to use can be obtained in a previous
+ * TLS/SSL connection using wolfSSL_get_session.
+ *
+ * This function rejects the session if it has been expired when this function
+ * is called. Note That this expiration check is wolfSSL specific and it brigns
+ * return code difference from OpenSSL.
+ *
+ * By default, wolfSSL_set_session returns WOLFSSL_SUCCESS on successfully
+ * setting the session, WOLFSSL_FAILURE on failure due to the session cache
+ * being disabled, or the session has expired.
+ *
+ * If you want to get the same return code when the session has expired,
```

Comment:
Let's reword this as "To match OpenSSL return code behavior when session is expired, define OPENSSL_EXTRA and WOLFSSL_ERROR_CODE_OPENSSL. This behavior will return WOLFSSL_SUCCESS even when the session is expired and rejected."

Reply:
Replaced the statements.